ui/src/utils/UtteranceApiService.js

```diff
@@ -76,7 +75,7 @@ export class UtteranceApiService {
         if (query.success) {
             return query.url;
         }
-        window.alert('Error: Unable to access database.');
```

Comment:
Don't we need to show toasts for these error conditions?

Reply:
a toast will be thrown by the RecordButton if this function returns false, but for now it's a generic one; should we get more specific?